composer.json

```diff
@@ -114,6 +114,7 @@
     "drupal/roleassign": "^2",
     "drupal/simple_oauth": "^5",
     "drupal/tfa": "^2",
+    "drupal/smtp": "^1",
```

Comment:
@baldwinlouie The item should be the alphabetical order:

```suggestion
    "drupal/smtp": "^1",
    "drupal/tfa": "^2",
```

Reply:
@naoi This has been updated.